plugins/plugins.pro

```diff
@@ -6,9 +6,9 @@
 TEMPLATE = subdirs
 
 CONFIG += debug_and_release
-SUBDIRS = link
+SUBDIRS = l4d2 link
```

Comment:
l4d2 only builds on Windows and Linux.

You should add a separate linux {} section and add l4d2 there as well.